scrapers/AdultEmpireCash.yml

```diff
@@ -0,0 +1,128 @@
+name: AdultEmpireCash
+# Sites ("Brands") from https://www.adultempirecash.com
+sceneByURL:
+  - action: scrapeXPath
+    scraper: sceneScraper
+    url:
+      - concoxxxion.com/
+      - elegantangel.com/
+      - filthykings.com/
+      - filthypov.com/
+      - forbiddenfruitsfilms.com/
+      - hornyhousehold.com/
+      - jayspov.net/
+      - jodiwest.com/
+      - kingsoffetish.com/
+      - latinoguysporn.com/
+      - lethalhardcore.com/
+      - manipulativemedia.com/
+      - mypervyfamily.com/
+      - pornstarstroker.com/ # aggregator site?
+      - reaganfoxx.com/
+      - severesexfilms.com/
+      - thirdworldxxx.com/
+      - touchmywife.com/
+      - westcoastproductions.com/
+
+  - action: scrapeXPath
+    scraper: movieScraper
+    url:
+      # - bangmytransass.com/ # Scenes published as movies?
+      # - blacktranswhackers.com/ # Scenes published as movies?
+      - concoxxxion.com/
+      - elegantangel.com/
+      - manipulativemedia.com/
+      - severesexfilms.com/
+      # - shemalestrokers.com/ # Scenes published as movies?
+      - thirdworldxxx.com/
+
+  # VR Sites
+  - action: scrapeXPath
+    scraper: vrSceneScraper
+    url:
+      - lethalhardcorevr.com/
+      - whorecraftvr.com/
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h1[@class="description"]/text()
+      Date:
+        selector: //div[@class="release-date"][1]/text()
+        postProcess:
+          - parseDate: Jan 02, 2006
+      Details: //div[@class="synopsis"]//text()
+      Tags:
+        Name: //div[@class="tags"]/a/text()
+      Performers:
+        Name: //div[@class="video-performer"]/a//text()
+      Studio:
+        Name:
+          selector: //div[@class="studio"]/span[2]
+          postProcess:
+            # Override studio name only when needed
+            - map:
+                Jay Rock Clips: Jay's POV
+                Filthy Kings Clips: FilthyPOV # filthykings.com + filthypov.com
```

Comment:
I'm unsure about this one:
Both sites list the same content.
The scene page lists the studio as "Filthy Kings Clips" for every scene, even ones that are "POV".
The two studios I can find are `FilthyPOV` and `FilthyTaboo`,
but no way to differentiate between them on these specific sites.

Reply:
Since the site also seems to call them "Niches",
I think the best solution would be to set the studio as the site name - `Filthy Kings`.